src/Zynga/Framework/Service/V2/Server/BaseTest.hh

```diff
@@ -0,0 +1,22 @@
+<?hh // strict
+
+namespace Zynga\Framework\Service\V2\Server;
+
+use Zynga\Framework\Testing\TestCase\V2\Base as TestCase;
+use Zynga\Framework\Service\V2\Interfaces\ServerInterface;
+use Zynga\Framework\Service\V2\Test\Server as TestServer;
+use Zynga\Framework\Service\V2\Test\Valid;
+
+
+class BaseTest extends TestCase {
+
+  public function testServerClass() : void {
```

Comment:
When writing tests, please try to make sure the test name signifies the condition you are looking for. 

For example, this test name tells me nothing about what the test is looking for. I could therefore not be quite sure if the assertion makes sense. Is there a better name for this test (and below)?

Something  like:

```
public function testServiceReturnsServerObject(): void {
```

Reply:
Why was this resolved? The function name hasn't changed nor was there a follow up comment.